cdrs-tokio/src/lib.rs

```diff
@@ -38,23 +38,13 @@
 //! balancing is preferred when dealing with multi-node clusters, otherwise simpler strategies might
 //! prove more efficient.
 
```

Comment:
https://github.com/krojew/cdrs-tokio/blob/f60a2f62f92f3c55795416eb81a2e9e78071ed06/cdrs-tokio/src/lib.rs#L11

Doc example on this line needs to be fixed

Reply:
fixed!